### DISCUSSION TITLE
Iraf in a container with module environment

### DISCUSSION THREAD

**Discussion by FSBoyden** [iraf-community/iraf#380] [Q&A]
Hi
I am trying to deploy IRAF in a large LAB environment and have found with other software suites that it works best to put the software suite in a container and load the binaries/apps using modules, i.e. module load iraf-2.17...
I host the software on a NFS repository which the dumb servers/terminals have access to.
During the module load process, the PATH and LD_LIBRARY_PATH values are set to the bin/ and lib/ locations. 
I have trouble getting ecl/cl to start and I presume it has to do with missing dependencies.

Q:
Has anyone done something like this?
Any recommendations on what links/paths I need to set or need to include?

p.s.
I really do not want to install it on every workstation, as it makes system management of the LAB a nightmare and just linking to a host is better and more efficient...

**Comment by olebole** [comment]
The main requirement here is that you should set the `iraf` environment variable accordingly, f.e. `export iraf=/net/server/packages/iraf/`. Note the trailing slash at the end. If you then have the path pointing to the IRAF binaries, everything should work. If not, please post the relevant error message.
If you use Debian or Ubuntu (or other derivatives) based lab workstations, you could just add the ["iraf" package](https://packages.debian.org/bookworm/iraf) to the list of installed packages. It is part of the distribution.
